Source/stores.cpp

```diff
@@ -161,14 +161,14 @@ void PrintStoreItem(ItemStruct *x, int l, text_color iclr)
 		if (x->_iSufPower != -1) {
 			PrintItemPower(x->_iSufPower, x);
 			if (sstr[0])
-				strcat(sstr, _(",  "));
+				strcat(sstr, ",  ");

```

Comment:
For now, I think it's best to keep them as some languages may not use comma to separate a series of nouns. In the belong run we should not use `strcat` either and instead have something more expressive like `"{power}, {power}"`;

Reply:
Fixed